### ISSUE TITLE
French SI Brochure feedback from BIPM (pages 4-20)

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/bipm-si-brochure#40]
page | comment
-- | --
4 | Instead of "La Brochure sur le SI", the text starts with "Ce document"
4 | the accents of the French words are all at wrong places: too high, too much on the left
4 | In the postal address at the bottom "Sèvres Cedex" should be on one line
6 | Bullet point 6: gamma and alpha should not be in italics
9 | Possible to bring item 5.4.8 to the previous page?
9 | Include the index in the table of contents
10 | In the title the accents on "à" and "édition" seem to be placed too high
12 | At the end of the first paragraph the reference to a page 5 is missing.
14 | 5. line of 2.2: "unité" should be written with the same font as"valeur numérique".
15 | List of constants: It seems that the text is written in bold, but the numbers are not.
15 | 2nd line above table 1: the negative exponents are not at the same height as the positive ones. Applies to many other places in the text
16 | 5. paragraph, line 4: an alpha is missing
16 | 6. paragraph: the isolated « a the end of line is not very nice
16 | 6. paragraph, the omega should be in italics (2 x)
16 | the isolated « a the end of line is not very nice
17 | 2.3, first phrase:  "unités  de  base" should be in italics (2 x)
17 | side note: the word "recommendations" should be in italics
17 | side note: replace "Article 5" with "chapitre 5"
18 | 3. paragraph: replace "Article 2.3.6" with "section 5.3.6"
18 | definition of the kilogram: the last two digits '15' of the numberical value should be separated by a small space, so that the digits are grouped by three.
18 | definition of the kilogram: the negative exponent is not at the same height as the positive
18 | last equation: 9 192 631 770 is missing in the equation
18 | last paragraph: the isolated » at the beginning of the line is not nice
19 | Definitionof the ampere: the symbol for the elementary charge, e, should be in italics
19 | line above first equation: the symbol for the elementary charge, e, should be in italics
19 | second paragraph from bottom: the symbol for π should not be in italics
19 | second paragraph from bottom: the « at the end of the line is not nice
19 | second paragraph from bottom: the symbol for the elementary charge, e, should be in italics
19 | last line: the symbol for π should not be in italics
20 | Definition of the kelvin: the negative and postitive exponents are not at the same height
20 | second equation: the decimals of 6,626 070 15 are wrongly grouped. They should be grouped as in this comments.
20 | paragraph below second equation: the unit J should not be separated from the value.
20 | last equation:  replace 273.5 with 273.15
20 | replace "article 5.4.1" with "section 5.4.1"

**Comment by Intelligent2013** [comment]
page | comment | to fix
-- | -- | --
4 | Instead of "La Brochure sur le SI", the text starts with "Ce document" | adoc
4 | the accents of the French words are all at wrong places: too high, too much on the left | xslt
4 | In the postal address at the bottom "Sèvres Cedex" should be on one line | adoc
6 | Bullet point 6: gamma and alpha should not be in italics | adoc
9 | Possible to bring item 5.4.8 to the previous page? | xslt
9 | Include the index in the table of contents | ?
10 | In the title the accents on "à" and "édition" seem to be placed too high | xslt
12 | At the end of the first paragraph the reference to a page 5 is missing. | adoc
14 | 5. line of 2.2: "unité" should be written with the same font as"valeur numérique". | adoc
15 | List of constants: It seems that the text is written in bold, but the numbers are not. | ?
15 | 2nd line above table 1: the negative exponents are not at the same height as the positive ones. Applies to many other places in the text | jEuclid?
16 | 5. paragraph, line 4: an alpha is missing | adoc
16 | 6. paragraph: the isolated « a the end of line is not very nice | adoc
16 | 6. paragraph, the omega should be in italics (2 x) | adoc
16 | the isolated « a the end of line is not very nice | adoc
17 | 2.3, first phrase:  "unités  de  base" should be in italics (2 x) | adoc
17 | side note: the word "recommendations" should be in italics | adoc
17 | side note: replace "Article 5" with "chapitre 5" | rb
18 | 3. paragraph: replace "Article 2.3.6" with "section 5.3.6" | rb
18 | definition of the kilogram: the last two digits '15' of the numberical value should be separated by a small space, so that the digits are grouped by three. | adoc
18 | definition of the kilogram: the negative exponent is not at the same height as the positive | jEuclid?
18 | last equation: 9 192 631 770 is missing in the equation | adoc
18 | last paragraph: the isolated » at the beginning of the line is not nice | adoc
19 | Definitionof the ampere: the symbol for the elementary charge, e, should be in italics | adoc
19 | line above first equation: the symbol for the elementary charge, e, should be in italics | adoc
19 | second paragraph from bottom: the symbol for π should not be in italics | adoc
19 | second paragraph from bottom: the « at the end of the line is not nice | adoc
19 | second paragraph from bottom: the symbol for the elementary charge, e, should be in italics | adoc
19 | last line: the symbol for π should not be in italics | adoc
20 | Definition of the kelvin: the negative and postitive exponents are not at the same height | jEuclid?
20 | second equation: the decimals of 6,626 070 15 are wrongly grouped. They should be grouped as in this comments. | adoc
20 | paragraph below second equation: the unit J should not be separated from the value. | adoc
20 | last equation:  replace 273.5 with 273.15 | adoc
20 | replace "article 5.4.1" with "section 5.4.1" | rb

**Comment by opoudjis** [comment]
rb: 

Not that BIPM have said this explicitly (and BIPM _must_ learn that they need to!!), but it appears that instead of using "Article" for clause references, BIPM uses "Chapître/Chapter" for first level clause references, and "Section" for subclauses.

**Comment by Intelligent2013** [comment]
@opoudjis  @manuel489 Please note that I've reassigned these issues to `adoc` and `rb`:

page | comment | to fix
-- | -- | --
4 | the accents of the French words are all at wrong places: too high, too much on the left | ~xslt~ adoc rb
10 | In the title the accents on "à" and "édition" seem to be placed too high | ~xslt~ adoc rb

It's a bug in FOP: https://issues.apache.org/jira/browse/FOP-2969

In the source xml there are a characters with accent/acutes as separated characters, i.e. for example:

In normal case - Latin Small Letter E With Acute has a hex code E9:
![изображение](https://user-images.githubusercontent.com/5559926/97786015-a10fd080-1bb9-11eb-9f6b-72644122fb28.png)

but in the source adoc and xml it encoded as e and hex code 301 (Combining Acute Accent)
`== Preface à la 9^e^ édition`
![изображение](https://user-images.githubusercontent.com/5559926/97786077-0cf23900-1bba-11eb-9aae-32ef3d91d1af.png)

![изображение](https://user-images.githubusercontent.com/5559926/97786117-6a868580-1bba-11eb-8b1d-07dcf27322c0.png)

These chars should be replaced to simple characters (i.e. as one char).

Comparing between old and updated (copied from word to xml directly) chars:
![изображение](https://user-images.githubusercontent.com/5559926/97786258-76267c00-1bbb-11eb-876b-78b794605b23.png)

![изображение](https://user-images.githubusercontent.com/5559926/97786342-f816a500-1bbb-11eb-99dd-41fe2b47a33c.png)

**Comment by opoudjis** [comment]
Wow! Yes, although Unicode processing should be dealing with that, this is indeed a problem. @manuel489 You are already editing the documents; can you fix this, or do you need me to?

**Comment by opoudjis** [comment]
Actually, don't worry: I'm going to apply Unicode Normalisation to the XML generation in standoc. @Intelligent2013 I will need you to confirm that it has worked. @manuel489  don't touch anything.

**Comment by opoudjis** [comment]
Turns out, it is any French text *I* have entered, in Mac Terminal: Mac Terminal is entering decomposed Unicode text. So that applies to the YAML tables and to the boilerplate. I will introduce normalisation anyway, since this can happen at any stage.

**Comment by anermina** [comment]
> page	comment	to fix
> 4	Instead of "La Brochure sur le SI", the text starts with "Ce document"	adoc
> 4	In the postal address at the bottom "Sèvres Cedex" should be on one line	adoc

These are not part of adoc file. This appears on the page right after the front page. @opoudjis? 

> 6	Bullet point 6: gamma and alpha should not be in italics	adoc
> 19	second paragraph from bottom: the symbol for π should not be in italics	adoc
> 19	last line: the symbol for π should not be in italics	adoc

We can fix this by applying some other font, such as `sf alpha` , `sf gamma` and `sf pi`. @ronaldtse would that be fine?
I didn't apply this yet since we also have a reverse case when italics are needed, but we don't have this kind of option, since something like `cc Omega` would produce the same result as `Omega`...

> 12	At the end of the first paragraph the reference to a page 5 is missing.	adoc

I added a cross-reference.

> 14	5. line of 2.2: "unité" should be written with the same font as"valeur numérique".	adoc
> 16	5. paragraph, line 4: an alpha is missing	adoc

Fixed.

> 16	6. paragraph: the isolated « a the end of line is not very nice	adoc
> 16	the isolated « a the end of line is not very nice	adoc

I removed blank spaces between `&nbsp;` and `«`/`»` symbols.

> 16	6. paragraph, the omega should be in italics (2 x)	adoc

As @manuel489 wrote in https://github.com/metanorma/bipm-si-brochure/issues/41#issuecomment-719969812, this is not fixable by markup, unless we insert a literal symbol. @ronaldtse?

> 17	2.3, first phrase: "unités de base" should be in italics (2 x)	adoc
> 17	side note: the word "recommendations" should be in italics	adoc

Fixed.

> 18	definition of the kilogram: the last two digits '15' of the numberical value should be separated by a small space, so that the digits are grouped by three.	adoc
> 20	second equation: the decimals of 6,626 070 15 are wrongly grouped. They should be grouped as in this comments.	adoc

As I understand, this will be solved in https://github.com/metanorma/metanorma-bipm/issues/44. We removed manual digits separation from adoc.

> 18	last equation: 9 192 631 770 is missing in the equation	adoc

Fixed.

> 18	last paragraph: the isolated » at the beginning of the line is not nice	adoc
> 19	second paragraph from bottom: the « at the end of the line is not nice	adoc

I removed blank spaces between `&nbsp;` and `«`/`»` symbols.

> 19	Definitionof the ampere: the symbol for the elementary charge, e, should be in italics	adoc
> 19	line above first equation: the symbol for the elementary charge, e, should be in italics	adoc
> 19	second paragraph from bottom: the symbol for the elementary charge, e, should be in italics	adoc
> 20	paragraph below second equation: the unit J should not be separated from the value.	adoc
> 20	last equation: replace 273.5 with 273.15	adoc

Fixed.

**Comment by opoudjis** [comment]
> 4 Instead of "La Brochure sur le SI", the text starts with "Ce document" adoc

That is deliberate by me, and I strongly urge us not to fix this. The boilerplate we are including in our generated documents is not intended just for the single document of the SI Brochure, but for all documents produced by BIPM. We cannot just insert the title of the document, because in French we don't know what the gender of the document title is (_La_ brochure, but _le_ rapport). And frankly, I do not believe that inserting the document title in the boilerplate is worth us adding a document title gender attribute.

> 4 In the postal address at the bottom "Sèvres Cedex" should be on one line adoc

It's boilerplate. Fixed.

**Comment by opoudjis** [comment]
>> 16 6. paragraph, the omega should be in italics (2 x) adoc
> As @manuel489 wrote in #41 (comment), this is not fixable by markup, unless we insert a literal symbol. @ronaldtse?

@anermina @manuel489 the italicisation of mathematical symbols will be addressed in https://github.com/metanorma/metanorma-standoc/issues/367

**Comment by opoudjis** [comment]
Updated PDF: https://github.com/metanorma/bipm-si-brochure/issues/24#issuecomment-723920534

**Comment by manuel489** [comment]
Based on the latest PDF provided by @opoudjis: [si-brochure-fr(12-01).pdf](https://github.com/metanorma/bipm-si-brochure/files/5622920/si-brochure-fr.12-01.pdf)


I've made comparisons between these three files: PDF reviewed by BIPM, latest PDF provided by Nick, and original PDF.
Below is the same table as in description but pointing out which issues are fixed and which are not yet.


page | comment | is issue fixed?
-- | -- | --
4 | Instead of "La Brochure sur le SI", the text starts with "Ce document" | no
4 | the accents of the French words are all at wrong places: too high, too much on the left | yes
4 | In the postal address at the bottom "Sèvres Cedex" should be on one line | yes
6 | Bullet point 6: gamma and alpha should not be in italics | yes
9 | Possible to bring item 5.4.8 to the previous page? | yes
9 | Include the index in the table of contents | no
10 | In the title the accents on "à" and "édition" seem to be placed too high | no
12 | At the end of the first paragraph the reference to a page 5 is missing. | yes
14 | 5. line of 2.2: "unité" should be written with the same font as"valeur numérique". | yes
15 | List of constants: It seems that the text is written in bold, but the numbers are not. | no
15 | 2nd line above table 1: the negative exponents are not at the same height as the positive ones. Applies to many other places in the text | yes
16 | 5. paragraph, line 4: an alpha is missing | yes
16 | 6. paragraph: the isolated « a the end of line is not very nice | yes
16 | 6. paragraph, the omega should be in italics (2 x) | no
16 | the isolated « a the end of line is not very nice | yes
17 | 2.3, first phrase:  "unités  de  base" should be in italics (2 x) | yes
17 | side note: the word "recommendations" should be in italics | yes
17 | side note: replace "Article 5" with "chapitre 5" | yes
18 | 3. paragraph: replace "Article 2.3.6" with "section 5.3.6" | yes
18 | definition of the kilogram: the last two digits '15' of the numberical value should be separated by a small space, so that the digits are grouped by three. | yes
18 | definition of the kilogram: the negative exponent is not at the same height as the positive | yes
18 | last equation: 9 192 631 770 is missing in the equation | yes
18 | last paragraph: the isolated » at the beginning of the line is not nice | yes
19 | Definition of the ampere: the symbol for the elementary charge, e, should be in italics | no
19 | line above first equation: the symbol for the elementary charge, e, should be in italics | no
19 | second paragraph from bottom: the symbol for π should not be in italics | yes
19 | second paragraph from bottom: the « at the end of the line is not nice | yes
19 | second paragraph from bottom: the symbol for the elementary charge, e, should be in italics | no
19 | last line: the symbol for π should not be in italics | yes
20 | Definition of the kelvin: the negative and postitive exponents are not at the same height | yes
20 | second equation: the decimals of 6,626 070 15 are wrongly grouped. They should be grouped as in this comments. | yes
20 | paragraph below second equation: the unit J should not be separated from the value. | yes
20 | last equation:  replace 273.5 with 273.15 | yes
20 | replace "article 5.4.1" with "section 5.4.1" | yes

**Comment by manuel489** [comment]
These are the issues that don't fulfill with BIPM requirements yet.

page | comment | is issue fixed?
-- | -- | --
4 | Instead of "La Brochure sur le SI", the text starts with "Ce document" | no
9 | Include the index in the table of contents | no
10 | In the title the accents on "à" and "édition" seem to be placed too high | no
15 | List of constants: It seems that the text is written in bold, but the numbers are not. | no
16 | 6. paragraph, the omega should be in italics (2 x) | no
19 | Definition of the ampere: the symbol for the elementary charge, e, should be in italics | no
19 | line above first equation: the symbol for the elementary charge, e, should be in italics | no
19 | second paragraph from bottom: the symbol for the elementary charge, e, should be in italics | no

**Comment by anermina** [comment]
> 19	Definition of the ampere: the symbol for the elementary charge, e, should be in italics	no
> 19	line above first equation: the symbol for the elementary charge, e, should be in italics	no
> 19	second paragraph from bottom: the symbol for the elementary charge, e, should be in italics	no

I'd just like to add that the last three issues were actually source adoc issues when BIPM reported them. In other words, this was the output:
![image](https://user-images.githubusercontent.com/55683568/100760149-bd3da200-33f1-11eb-9b14-76b185276fc5.png)

These issues have been solved and now we have the output where `e` is in the same font as in other parts of the text, for which BIPM didn't add any notes. I believe they are fine with this appearance of the letter `e`. Yellow highlighted (corrected) is the same as red underlined (initial output).
![image](https://user-images.githubusercontent.com/55683568/100760582-32a97280-33f2-11eb-904c-96c976611ef5.png)

**Comment by manuel489** [comment]
Wow! Thanks @anermina for double-checking!
Hmm, from that point of view, yes I think you're right 😃.

The reason I marked it as not-fixed issue is because I compared it with the font of the original file and they are different:

![Capture](https://user-images.githubusercontent.com/33627611/100764003-3f1ad480-33cc-11eb-8448-08722167fe5c.PNG)

*left: generated, right: original*

But yeah, that is probably because we are using a different set of fonts. Which in this case, the difference between *italics* and *not italics* is a little bit more subtle (but definitely noticeable):

![Capture2](https://user-images.githubusercontent.com/33627611/100764925-455d8080-33cd-11eb-8887-fc081b4d553e.PNG)

*above: italics, bottom: not italics*

**Comment by manuel489** [comment]
So, discarding the above issues the table results in:

page | comment | is issue fixed?
-- | -- | --
4 | Instead of "La Brochure sur le SI", the text starts with "Ce document" | no
9 | Include the index in the table of contents | no
10 | In the title the accents on "à" and "édition" seem to be placed too high | no
15 | List of constants: It seems that the text is written in bold, but the numbers are not. | no
16 | 6. paragraph, the omega should be in italics (2 x) | no

**Comment by anermina** [comment]
No problem @manuel489 , I just remembered seeing this in some of the previous checks :)

**Comment by opoudjis** [comment]
p. 4: https://github.com/metanorma/bipm-si-brochure/issues/40#issuecomment-721114787 . I am currently declining to implement, as I want strongly boilerplated text.

p. 9: There is currently no index anyway; https://github.com/metanorma/metanorma-bipm/issues/67 is the task to create it, and it is a big undertaking

p. 10. The accents are supposed to have been fixed, by me applying Unicode Normalisation to the text generation; will need to investigate what has happened.

p. 15, 16: to investigate

**Comment by opoudjis** [comment]
p. 10, clearly I missed this file. Fixed.

**Comment by opoudjis** [comment]
p. 15. Boldface applied to text does not apply to the MathML it contains; so any boldface has to be indicated explicitly on the AsciiMath: the MathML processor simply ignores the formatting of its context. (I don't think that's the case in Word, but then, Word is not an exemplar of mathematical word processing.)

Unfortunately this means you're also going to have to insert `bii()`, bold italic, since the override is going to wipe out any italics already there. 

Moreover, nesting of font styles tends not to work.

So you get something disgusting to data enter like:

````
* *la fréquence de la transition hyperfine de l’état fondamental de l’atome de césium 133 non perturbé, stem:[bb(Delta) bii(nu)_bb("Cs")], est égale à stem:[bb(9192631770) " " bb(H)bb(z)],*
* *la vitesse de la lumière dans le vide, stem:[bii(c)], est égale à stem:[bb(299792458) " " bb(m)//bb(s)],*
* *la constante de Planck, stem:[bii h], est égale à stem:[bb(6.62607015 xx 10^{-34}) " " bb(J) " " bb(s)],*
* *la charge élémentaire, stem:[bii e], est égale à stem:[bb(1.602176634 xx 10^{-19}) " " bb(C)],*
* *la constante de Boltzmann, stem:[bii k], est égale à stem:[bb(1.380649 xx 10^{-23}) " " bb(J)//bb(K)],*
* *la constante d’Avogadro, stem:[bii N_"A"], est égale à stem:[bb(6.02214076 xx 10)^bb(23) " " bb(mol)^{-1})],*
* *l’efficacité lumineuse d’un rayonnement monochromatique de fréquence stem:[bb(540 xx 10^12) " " bb(H)bb(z)], stem:[bii(K_{"cd"})], est égale à stem:[bb(683) " " bb(lm)//bb(W)],*
````

AND, any such overrides of style are themselves going to be overridden by the styles we insert to deal with upright Greek characters. So we can't make Greek characters boldface without more postprocessing.

AND in JEuclid (PDF), boldface is being ignored for exponents.

As a result, what I'm getting with the data entry above is:

PDF:

<img width="726" alt="Στιγμιότυπο 2020-12-04, 23 59 11" src="https://user-images.githubusercontent.com/1911039/101166684-c5fdd480-368c-11eb-952d-c13758e1cea6.png">

HTML:

<img width="965" alt="Στιγμιότυπο 2020-12-05, 00 00 42" src="https://user-images.githubusercontent.com/1911039/101166793-f6de0980-368c-11eb-9a48-546c31882568.png">


I have no idea why random letters H and W are not being bolded by jEuclid.

This is enough of a disaster to push back on the use of boldface for emphasising mathematical text, which is frankly ugly to begin with. I do not think we should be proceeding with this.

**Comment by opoudjis** [comment]
p. 16. 

There are two different things going on here.

First: If this were a `<mi>` operator, as we would expect, it would be rendered in italics. It is being rendered as a `<mo>` instead by the AsciiMath converter.

Second: font commands will work in the general case; e.g. `stem:[ii Omega]` instead of `stem:[Omega]`. However, any such font commands should override the font commands introduced for maths localisation. For example, we expect `stem:[Omega]` to render `<mi>&Omega;</mi>`; that is localised for French typesetting as `<mi mathvariant="normal">&Omega;</mi>`. We expect `stem:[ii Omega]` to be respected, and to cancel the override of `mathvariant="normal"`.

This arguably annuls any font nesting we might try to introduce; but we have just seen that font nesting in MathML does not work anyway.

Addressing the second, and inviting editors to insert explicit font overrides in cases like this. The first needs further investigation of the asciimath gem, but might not be tractable.

**Comment by opoudjis** [comment]
First issue: Omega is being parsed by AsciiMath as an operator, because, well, it is an operator in mathematics: https://en.wikipedia.org/wiki/Omega#The_symbol_Ω_(uppercase_letter) ; e.g. 

> In number theory, Ω(n) is the number of prime divisors of n (counting multiplicity)

Greek Letters that are treated by AsciiMath as operators are: https://github.com/asciidoctor/asciimath/blob/7b48295ae5c5bfc3f4679daa9ca322f69aec1931/lib/asciimath/markup.rb

Γ, Δ, Θ, Λ, Ξ, Π, Σ, Ω

The French localisation of mathematical variables is not going to catch operators `<mo>`. So we have a problem with trying to do fine semantics with AsciiMath.

We can override this behaviour by postprocessing, to say that if any of those operator letters is not followed by parentheses in the same MathML expression, they should be switched to `<mi>`. I'm wary of spending too much time fixing these, but the blanket assumption that those capital Greek letters are mathematical operators (which are always upright) is inapplicable in non-mathematical use of Greek letters, such as in physics.

This is very convoluted; let me summarise it.

| LETTER                         | AsciiMath RENDERING | Default Rendering Style | French Rendering Style |
| ------------- |-------------| -----|-----|
| lowercase Roman a      | mi                                   | italic                                | italic                                 |
| uppercase Roman A      | mi                                   | italic                                | upright                                 |
| lowercase Greek alpha | mi | italic | upright | 
| uppercase Greek alpha | mi | italic | upright | 
| uppercase Greek omega | mo (INCORRECT) | upright | upright | 

We have the unfortunate situation that:

* Capital omega is being treated as an operator, which renders it as upright
* Capital omega should being treated as a variable, which still renders it under French rules as upright
* The expected rendering in the document in this case is as italics, which violates French rules, and observes English rules instead.

The markup `stem:[ii Omega]` will address the rendering in either case, and the semantics in my opinion is not a priority to fix, especially if Units markup is going to be revisited anyway. But the operator treatment of Greek letters is a gotcha in Asciimath.

**Comment by manuel489** [comment]
Ok, italic markup for `Omega` symbols and `M` operators are applied in https://github.com/metanorma/bipm-si-brochure/pull/59

**Comment by manuel489** [comment]
> I have no idea why random letters H and W are not being bolded by jEuclid.
> This is enough of a disaster to push back on the use of boldface for emphasising mathematical text, which is frankly ugly to begin with. I do not think we should be proceeding with this.

@opoudjis, just to confirm, should we proceed with boldface markup in math environment?

**Comment by manuel489** [comment]
Two remaining issues in this thread:

page | comment | issue
-- | -- | --
4 | Instead of "La Brochure sur le SI", the text starts with "Ce document" | will not fix, https://github.com/metanorma/bipm-si-brochure/issues/40#issuecomment-736907522
9 | Include the index in the table of contents | not fixed
10 | In the title the accents on "à" and "édition" seem to be placed too high | fixed
15 | List of constants: It seems that the text is written in bold, but the numbers are not. | not fixed
16 | 6. paragraph, the omega should be in italics (2 x) | fixed

**Comment by manuel489** [comment]
@opoudjis , there is another issue that weren't before and I missed to report.

An additional entry in the index, `Références`, that is not in the original document:

![Capture1](https://user-images.githubusercontent.com/33627611/101206085-0174b080-3645-11eb-9916-27274f273626.PNG)


And it takes a whole page for itself.

**Comment by opoudjis** [comment]
@manuel489 The references are correctly deleted from the documents within the bilingual brochure folder.

To explain:

* A reference of a document to another document *must* result in a bibliography
* A reference from a document to another document within a collection results in that reference being hidden—because citations are replaced with cross-references
* So those references are hidden in the documents generated as part of collection processing—the documents generated within the bilingual-brochure folder.
* OTOH those references are NOT hidden in the French brochure when it is generated, beforehand, as a self-standing document. 
* _And that behaviour is correct_. It is the height of absurdity for a document to reference another document only by page number, when it is being treated as standalone. By definition. The surprise here is only in that BIPM have not generated these documents as standalone.

Because the references are being deleted in the collection brochure, there is nothing for me to fix.

**Comment by manuel489** [comment]
Ok @opoudjis, thank you for your clarification.

**Comment by anermina** [comment]
Indexing is addressed in metanorma/metanorma-bipm#67. Closing this ticket.
